Tools/Pipeline/Common/AssemblyAttributes.cs

```diff
@@ -81,6 +81,14 @@ public static string AssemblyCompany
 				return ((AssemblyCompanyAttribute)attributes[0]).Company;
 			}
 		}
+
+        public static string[] Developers
+        {
+            get
+            {
+                return new string[] { "cra0zy (Harry)", "dellis1972 (Dean Ellis)", "dneelyep (Daniel Neel)", "emarcotte (Eugene Marcotte)", "JamesLupiani (James Lupiani)", "jamesford42 (James Ford)", "KonajuGames (Steve 'Sly' Williams)", "mgarstenauer (Martin Garstenauer)", "nightst4r (Mark)", "nkast (Nikos Kastellanos)", "polsys (Petri Laarne)", "RayBatts (Ray Batts)", "Shqrdx (Yuri Roubinsky)", "thiagoabreu (Thiago 'Jedi' Abreu)", "tomspilman (Tom Spilman)" };
```

Comment:
I like the sentiment. But if we are going to list people who have
contributed perhaps we should just setup a contributors page/section on the
main monogame.net site and list everyone who has contributed over the
years. We can easily generate the data for a page with something like

git log --format='%aN <%aE>' | awk '{arr[$0]++} END{for (i in arr){print
arr[i], i;}}' | sort -rn


 Although we would probably want to only print the username / Name rather
than email addressed (this is what this command does :) )

On 28 April 2015 at 21:02, Tom Spilman <notifications@github.com> wrote:

> In Tools/Pipeline/Common/AssemblyAttributes.cs
> <https://github.com/mono/MonoGame/pull/3777#discussion_r29282298>:
>
> > @@ -81,6 +81,14 @@ public static string AssemblyCompany
> >  				return ((AssemblyCompanyAttribute)attributes[0]).Company;
> >  			}
> >  		}
> > +
> > +        public static string[] Developers
> > +        {
> > +            get
> > +            {
> > +                return new string[] { "cra0zy (Harry)", "dellis1972 (Dean Ellis)", "dneelyep (Daniel Neel)", "emarcotte (Eugene Marcotte)", "JamesLupiani (James Lupiani)", "jamesford42 (James Ford)", "KonajuGames (Steve 'Sly' Williams)", "mgarstenauer (Martin Garstenauer)", "nightst4r (Mark)", "nkast (Nikos Kastellanos)", "polsys (Petri Laarne)", "RayBatts (Ray Batts)", "Shqrdx (Yuri Roubinsky)", "thiagoabreu (Thiago 'Jedi' Abreu)", "tomspilman (Tom Spilman)" };
>
>  Since you don't like it, I'll remove it immediately.
>
> Wait... lets see what others think first.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/MonoGame/pull/3777/files#r29282298>.
>